test/puppetlabs/puppetdb/admin_clean_test.clj

```diff
@@ -66,17 +66,22 @@
     (let [pdb (get-service *server* :PuppetDBServer)
           orig-clean cli-svc/clean-up
           in-clean (CyclicBarrier. 2)
-          test-finished (CyclicBarrier. 2)]
+          test-finished (CyclicBarrier. 2)
```

Comment:
The changes in this namespace aren't necessarily stockpile/activemq related, but @rbrw and I fixed a race condition and knowing that this PR was coming, put the commits here instead.